src/components/FilterSidebar/components/QAFilter.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { FilterData } from '@/context/FilterContext/types'; // adjust if needed
+import { render, screen, userEvent } from '@/utils/renderers';
+
+import { QAFilter } from './QAFilter'; // adjust the import based on your file structure
```

Comment:
Just some small comments that could be confusing

Reply:
I'll remove these.